Add aws-k8s-1.21, vmware-k8s-1.21 variants (#1612)
bottlerocket-os/bottlerocket: README.md

> Another variant we have in preview is designed to be a Kubernetes worker node in VMware:

Reviewer: ```suggestion
Other variants we have in preview are designed to be Kubernetes worker nodes in VMware:
```
